.mobiledgex/appinsts.yml

```diff
@@ -0,0 +1,29 @@
+- appinst:
+    key:
+      cluster_inst_key:
+        cluster_key:
+          name: autoclusteredgemultiplay
```

Comment:
will this always be the same cluster name regardless of where it is deployed?

Reply:
autoclusteredgemultiplay we have this cluster in the 3 regions